### PR TITLE
Open trace automatically if it is a Perfetto trace

### PR DESCRIPTION
**Pull request by dpallotti**
As per b/143711475, when a Perfetto trace is successfully taken, in almost every
case the user would want to click "Open Trace". This opens the trace
automatically, removing the extra step.

Bug: b/143711475

### CODE REVIEW

**Comment by dpallotti** [comment]
Not sure if this should be handled by the TraceProgressDialog like I did or by showTracingDialog directly.

**Comment by dpallotti** [comment]
Also, if we think that there are relevant cases where users might not want the Perfetto trace to be opened automatically, we could add an option to open it automatically or not.

**Comment by AWoloszyn** [comment]
See b/142317175

File: gapic/src/main/com/google/gapid/views/TracerDialog.java [hunk 1]
```diff
@@ -961,6 +970,9 @@ public boolean successful() {
     public void onProgress(StatusResponse progress) {
       status = progress;
       update();
+      if (progress.getStatus() == Service.TraceStatus.Done) {
```
Comment (dpallotti) [inline gapic/src/main/com/google/gapid/views/TracerDialog.java]: I obviously forgot to check this.openOnDone here

**Pull request closed without merging**